core/client/src/main/java/alluxio/client/block/AlluxioBlockStore.java

```diff
@@ -43,22 +43,29 @@
   private static AlluxioBlockStore sClient = null;
 
   /**
-   * @return a new instance of Alluxio block store
+   * @return an instance of the Alluxio block store
```

Comment:
updated, thanks. Wishful thinking about the future where this isn't a singleton :)